eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMapIterable.java

```diff
@@ -172,6 +172,11 @@ default V getOrDefault(Object key, V defaultValue)
         return this;
     }
 
+    default void putAllMapIterable(MapIterable<? extends K, ? extends V> mapIterable)
```

Comment:
@donraab the return type here is a `void` should we return a `MutableMapIterable` instead of void, i.e. return a `this`?
The way I look at it is that, if we return a `MutableMapIterable` it never hurts, but if we return a void, reassigning in user code is impossible. 
I prefer returning `this`. Thoughts?
Any case, we will take this in as `void` and fix it in a next PR.

Reply:
Revisiting my point from earlier, I think let us have `putAllMapIterable` return a `void` because we are adding `withMapIterable` which is returns `this`.

Reply:
I think `putAllMapIterable` should be symmetric with `putAll`. What does `putAll` return?

Reply:
`void`